src/platform/windows/mod.rs

```diff
@@ -63,7 +64,7 @@ impl MediaControls {
     /// Attach the media control events to a handler.
     pub fn attach<F>(&mut self, event_handler: F) -> Result<(), Error>
     where
-        F: Fn(MediaControlEvent) + Send + 'static,
+        F: Fn(MediaControlEvent) + Send + Sync + 'static,
```

Comment:
Why was this Sync clause added? The interface must remain `F: Fn(MediaControlEvent) + Send + 'static` on all systems.

Reply:
Refer to https://github.com/microsoft/windows-rs/commit/afe32525c22209aa8f632a0f4ad607863b51796a#diff-8e5c79df437470375e1139a935230234932ca20cb0fba4dcc8f664b53e7e0b34R4416, The event_handler need to be `Send`, so `F` need to be `Send` and `Sync` (https://github.com/rust-lang/rust/blob/f8588549c3c3d45c32b404210cada01e2a45def3/library/alloc/src/sync.rs#L241).

Reply:
Fixed.